From d44efad798c313288f4b136b08e88ad580880673 Mon Sep 17 00:00:00 2001
From: Claudio Saavedra <csaavedra@igalia.com>
Date: Tue, 15 Jun 2010 15:47:18 +0000
Subject: Fix build without libjpeg

AM_CONDITIONAL calls should be called always, otherwise configure
gets confused.

Fixes bug #621616.
---
diff --git a/configure.ac b/configure.ac
index 3631dd7..8f1e713 100644
--- a/configure.ac
+++ b/configure.ac
@@ -227,7 +227,6 @@ int main(int c, char**v) { return 0; }
 			   have_libjpeg_80=yes],
 		          [AC_MSG_RESULT(no)
 		           have_libjpeg_80=no])
-        AM_CONDITIONAL(HAVE_LIBJPEG_80, test "x$have_libjpeg_80" = xyes)
 
       else
           AC_MSG_WARN(*** JPEG loader will not be built (JPEG header file not found) ***)
@@ -243,6 +242,7 @@ int main(int c, char**v) { return 0; }
   fi
 AC_SUBST(LIBJPEG)
 AM_CONDITIONAL(ENABLE_JPEG, test x$have_jpeg = xyes)
+AM_CONDITIONAL(HAVE_LIBJPEG_80, test "x$have_libjpeg_80" = xyes)
 
 # ****************
 # D-Bus activation
--
cgit v0.8.3.1
